app-config.yaml

```diff
@@ -150,8 +150,15 @@ catalog:
   rules:
     - allow: [Component, System, Group, Resource, Location, Template, API]
   locations:
-    - type: url
-      target: https://github.com/redhat-developer/rhdh/blob/main/catalog-entities/all.yaml
+    - type: file
+      target: ../../catalog-info.yaml
```

Comment:
when this app-config is being parsed, is the CWD in `./packages/app`? Just making sure that it's okay to put  `../../` in a file that is already on the root repo directory

Reply:
Hey, the default is `packages/backend`, the default Backstage scaffolder contains a comment like this:

https://github.com/christoph-jerolimov/backstage-history/blob/main/app-config.yaml#L80-L95

That was removed here. I think this app-config.yaml is only used in local development. Production installation will not see all these "test entities".

But it's maybe worthful to check that again with someone that is more involved into the build process.